scrapers/Staxus.yml

```diff
@@ -0,0 +1,382 @@
+name: "Staxus"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title:
+        selector: //title/text()
+        postProcess:
+          - replace:
+              - regex: 'Staxus Scene - '
+                with: ""
+      Date:
+        selector: //div[@class='video-details'][1]
+        postProcess:
+          - replace:
+              - regex: "Added: "
+                with: ""
+          - parseDate: 02/January/2006
+      Tags:
+        Name: //p/a[@href[contains(., "category.php")]]/text()
+      Performers:
+        Name:
+          selector: //div[@class="video-descr__model-item"]/p/a/text()
+      Details:
+        selector: //div[@class="video-descr__content"]/p/text()
+        concat: "\n"
+      Image:
+        selector:  //script[@type="application/ld+json"][contains(.,'"thumbnailUrl":')][1]
+        postProcess:
+          - replace:
+              - regex: '^.*thumbnailUrl": "'
+                with: ""
+              - regex: '.jpg.*$'
+                with: ".jpg"
+              - regex: 'contentthumbs'
+                with: "content/contentthumbs"
+      Studio:
+        Name:
+          fixed: Staxus
+
+  performerScraper:
+    performer:
+      Name: //div[@class="model-header-details"]/h4/text()
+      Gender:
+        fixed: Male
+      Height:
+        selector: //p[@class="flex-item"]/b[text()="Height:"]/following-sibling::span[1]/text()
+        postProcess:
+          - replace:
+              - regex: 'cm.*/.*'
+                with: ""
+              - regex: '^.*/'
+                with: ""
```

Comment:
For the height we want to keep only the digits ( we save in metric units in the db)
```yaml
        postProcess:
          - replace:
              - regex: 'cm.*'
                with: ""
              - regex: '^.*/(\s)?'
                with: ""
```
the above seems to work better since in most cases i found the cm were after the `/`